spec/fuzzy-finder-spec.coffee

```diff
@@ -585,6 +585,20 @@ describe 'FuzzyFinder', ->
           expect(atom.workspace.panelForItem(projectView).isVisible()).toBe false
           expect(inputView).toHaveFocus()
 
+  describe "when the filter text contains an absolute path", ->
+    it "it matches the given path", ->
```

Comment:
thanks for the review, 
not sure about your comment, you suggest to replace
```coffee
describe "when the filter text contains an absolute path", ->
   it "it matches the given path", ->
```
by 
```coffee
   it "it matches the given path", ->
```
?